install-odoo-linux.md

```diff
@@ -0,0 +1,257 @@
+1) ajouter un utilisateur odoo

```

Comment:
This should be markdownized.

Reply:
This could be renamed as `install-odoo-linux-server.md` because it contains the process to install odoo on a linux server, not for a development environment.

Reply:
Ok, next time someone sets up a linux dev environment, it'd be nice to have a tutorial.

Reply:
I'll send my own here when I found time to do it. ;)